app/assets/stylesheets/report.scss

```diff
@@ -0,0 +1,32 @@
+.report {
+  background: #fff;
+  border-radius: 2px;
+  display: block;
+  float: left;
+  min-height: 300px;
+  margin: 1rem;
+  position: relative;
+  width: 100%;
+  box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.12)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.24)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by a single space